packages/scss/src/components/form/index.scss

```diff
@@ -56,7 +56,9 @@
 	}
 
 	&:has(.textField-input-value[aria-invalid='true']),
-	&:has(.timePicker-fieldset-group-textfield-input[aria-invalid='true']) {
+	&:has(.timePicker-fieldset-group-textfield-input[aria-invalid='true']),
+	&:has(.simpleSelect-input[aria-invalid='true']),
+	&:has(.multipleSelect-displayer-search[aria-invalid='true']) {
```

Comment:
`[aria-invalid='true']` without a prefix would be too permissive?

Reply:
I wonder... It could be a good approach but I'm a bit afraid of specific cases and it would require extra tests.